docs/Learning-Environment-Design-Agents.md

```diff
@@ -798,10 +796,9 @@ be called independently of the `Max Step` property.
       effective size of the vector observation being passed to the Policy being:
       _Space Size_ x _Stacked Vectors_.
   - `Vector Action`
-    - `Space Type` - Corresponds to whether action vector contains a single
-      integer (Discrete) or a series of real-valued floats (Continuous).
-    - `Space Size` (Continuous) - Length of action vector.
-    - `Branches` (Discrete) - An array of integers, defines multiple concurrent
+    - `Continuous Action Size` - The number of continuous actions that the Agent
```

Comment:
Should we say 'concurrent continuous actions'

Reply:
Updated